pkg/infra/pulumi_aws/iac/api_gateway.ts

```diff
@@ -162,8 +160,9 @@ export class ApiGateway {
                 },
             },
             {
-                dependsOn: [vpcLink],
                 parent: api,
+                replaceOnChanges: ['*'],
+                deleteBeforeReplace: true,
```

Comment:
We already do this for API Gateways here: https://github.com/klothoplatform/klotho/pull/208/files#diff-86a7f02a80cba7c191b1ed596a10dcceb360abb88a53be60c32bd9a195480d41R321-R344

We may want to fix it for this case too, but let's at least fix the tests then prioritize the downtime.

Reply:
that link takes me right to the addition of these lines so a little confused where its. supposed to take me to

Reply:
https://github.com/klothoplatform/klotho/blob/main/pkg/infra/pulumi_aws/iac/api_gateway.ts#L321-L345

Guess it doesn't like linking to places outside of the diff context.

Reply:
ah i see, yeah i mean if this works then good enough for now, but we likely want to figure out a good way to solve this moving forward because it would be bad for a production env